### PR TITLE
wrote test for component

### PR DESCRIPTION
**Pull request by AllStackDev1**
Closes: #1738

### CODE REVIEW

File: expo-zustand-styled-components/src/components/RepoActionButtons/CountButtonGroup.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react-native';
+import CountButtonGroup from './CountButtonGroup';
+import { Text } from 'react-native';
+
+describe('CountButtonGroup', () => {
+
+  afterEach(() => {
+    jest.resetAllMocks();
```
Comment (honzikec) [inline expo-zustand-styled-components/src/components/RepoActionButtons/CountButtonGroup.spec.tsx:9]: Is this needed? I don't see any mocks in this test suite 🤔

**Review by honzikec — COMMENTED** [review]
Looks good, just a couple of comments...

File: expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+import RepoActionButtons from '../RepoActionButtons';
+import { useRepoInfoStore } from '../../hooks/stores';
+
+jest.mock('../../hooks/stores', () => ({
```
[the remark this answers is not in the document]
Comment (AllStackDev1) [reply expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.spec.tsx:6]: Hmm, we only have one test case here, so no need of resetting.

**Review by honzikec — APPROVED** [review]
LGTM 🚢

**Pull request merged**